app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb

```diff
@@ -14,6 +14,14 @@
         subtitle: "<%= object.subtitle %>",
         axisTitle: "<%= object.axis_title %>",
         pointStart: <%= object.point_start %>,
+        xCategories: <%= object.x_categories.any? ? object.sanitized_x_categories : "undefined" %>,
+        xMin: <%= object.x_min %>,
+        xMax:<%= object.x_max %>,
+        xTickInterval: <%= object.x_tick_interval %>,
+        yCategories: <%= object.y_categories.any? ? object.sanitized_y_categories : "undefined"  %>,
```

Comment:
We just push this logic up into the view object?

Reply:
I think that all of this chart object could go into the view model.